tests/test_lab_evaluate.py

```diff
@@ -494,3 +494,47 @@ def test_invalid_tasks_dir(cli_runner: CliRunner):
     )
     assert "Tasks dir not found:" in result.output
     assert result.exit_code != 0
+
+
+def test_invalid_model_path_mmlu(cli_runner: CliRunner, tmp_path):
+    test_dir = tmp_path / "test"
+    test_dir.mkdir()
+    result = cli_runner.invoke(
+        lab.ilab,
+        [
+            "--config=DEFAULT",
+            "model",
+            "evaluate",
+            "--benchmark",
+            "mmlu",
```

Comment:
you could probably expand both test cases with variants that would cover both `_branch` and non-`_branch` options. (e.g. with `pytest.mark.parametrize`).

Reply:
I may do this in a followup if that's alright; want to try to get this in today and have some other items to do as well